### ISSUE TITLE
The Location pane doesn't draw a line connecting moved differences by default

### ISSUE THREAD

**Issue by Shun-Tsukamoto** [WinMerge/winmerge#498]
I found no line(s) appears in the Location pane when you have moved block(s).

Tested with:
Winmerge 2.16.9.0-x64
Windows 10 Home 64bit 20H2 19042.630

![moved block](https://user-images.githubusercontent.com/74853405/100758145-86d44700-3432-11eb-9c64-427b748d135a.PNG)

**Comment by sdottaka** [comment]
To draw connecting lines, right-click on the location pane and select All Moved Blocks or Moved Block for Current Diff menu item.

![image](https://user-images.githubusercontent.com/98126/100806627-236b0900-3474-11eb-9023-605774b306bd.png)

**Comment by Shun-Tsukamoto** [comment]
Thank you for your prompt reply! Now I know how to turn on the "connection line".
My original question has been resolved but I found small room to improve this behavior to make it easy to use.

I expected the "connection line" is drawn when you check the "Enable moved block detection" in the option. There's no clear instruction on the user's manual to turn it on.

Suggestion
"All Moved Block" is checked by default when you check the  "Enable moved block detection".
